pkg/daemon/update.go

```diff
@@ -124,6 +126,26 @@ func (dn *Daemon) updateOSAndReboot(newConfig *mcfgv1.MachineConfig) error {
 	return dn.reboot(fmt.Sprintf("Node will reboot into config %v", newConfig.GetName()))
 }
 
+// isUpdating returns true if the MCD is actively applying an update
+func (dn *Daemon) catchIgnoreSIGTERM() {
+	if dn.installedSigterm {
+		return
+	}
+
+	termChan := make(chan os.Signal, 1)
+	signal.Notify(termChan, syscall.SIGTERM)
+
```

Comment:
installedSigterm needs to be set... likely here?

Reply:
Thanks, fixed!

(Though currently it doesn't matter as the MCD doesn't ever call this function more than once in the same process...this was an attempt to optimize for a future where we might)